.circleci/ci/src/jobs/helm/job-release-helm.ts

```diff
@@ -54,12 +53,17 @@ git config --global user.email "\${GIT_USER_EMAIL}"`,
       new reusable.ReusedCommand(orbs.helm.commands['install_helm_client']),
     ];
 
+    steps.push(new commands.Checkout());
+
     if (!environment.isDryRun) {
       steps.push(
+        new commands.Run({
+          name: `Checkout tag ${apimVersion}`,
+          command: `git checkout ${apimVersion}`,
+        }),
         new commands.Run({
           name: 'Update Chart and App versions',
-          command: `sed "0,/version.*/s/version.*/version: ${apimVersion}/" -i helm/Chart.yaml
-sed "0,/appVersion.*/s/appVersion.*/appVersion: ${apimVersion}/" -i helm/Chart.yaml`,
+          command: `sed "0,/appVersion.*/s/appVersion.*/appVersion: ${apimVersion}/" -i helm/Chart.yaml`,
```

Comment:
I think that we have to update `version` and `appVersion` and not only the last one.
Do I miss something ?